README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# CoviData Metrics Webapp
+This project is a mobile web applicaation used to check a list of COVID-19 metrics.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![HomePage](./src/images/Screenshot1.png)          ![Details Page](./src/images/Screenshot2.png)
 
-## Available Scripts
+## Built With
+- JavaScript/React
+- Redux
+- React Router
+- JSX
+- CSS
+  
+## Setup
+- Get the link of the repository: [https://github.com/chaw-bot/react-capstone-project](https://github.com/chaw-bot/react-capstone-project)
+- Clone it as [git@github.com:chaw-bot/react-capstone-project.git](git@github.com:chaw-bot/react-capstone-project.git) on a Terminal
+- Usage
+- Run npm install on a Terminal to install the modules
+- Run npm start on a Terminal to start the server
 
-In the project directory, you can run:
+## Links
+- Click [here](https://covidataworldwide.netlify.app/) to view it in the browser.
+- [Loom video](https://www.loom.com/share/79bd6f0e21be43059a2b2b45ab1d1455)
+  
+## Authors
```

Comment:
- [ ] For proper documentation let's change 'Authors' to 'Author' since this is actually a solo project